### PR TITLE
Second styling pass

### PR DESCRIPTION
**Pull request by shannonwells**
Based on feedback from Heather

## Verification
1. Download and run `npm run dev`
1. Match it up with the shared design from Figma
Screenshot 2023-08-22 at 10 31 33 AM
Screenshot 2023-08-22 at 10 32 42 AM

### CODE REVIEW

File: src/lib/assets/logo.png
Comment (wilwade) [inline src/lib/assets/logo.png:1]: I can get you the svg if you would prefer it.

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -9,49 +9,59 @@
...
+<div class="mt-8 text-white">
   <form id="setupForm">
-    <Connect />
-    <div class={connected ? '' : 'hidden'}>
+    <Connect/>
+    <div class:hidden={!connected} class="mt-8">
       <KeySelection
         component="TransactionSigningKey"
-        selectLabel="Choose a Transaction Signing Address"
+        selectLabel="Choose a Wallet Address"
```
Comment (wilwade) [inline src/routes/+page.svelte:58]: In the e2e test this also needs to update: `const signer = screen.getByLabelText('Choose a Transaction Signing Address');`

**Review by wilwade — APPROVED** [review]
🏆 Ship it!

File: src/lib/assets/logo.png
Comment (shannonwells) [reply src/lib/assets/logo.png:1]: it's fine.

**Pull request merged**